doc/source/serve/dev-workflow.md

```diff
@@ -61,10 +61,6 @@ serve run local_dev:graph
 
 The `serve run` command blocks the terminal and can be canceled with Ctrl-C. Typically, `serve run` should not be run simultaneously from multiple terminals, unless each `serve run` is targeting a separate running Ray cluster.
 
-:::{note} 
```

Comment:
Why are we removing this note? Do users no longer need to start Ray in the relevant directory?